.changesets/docs_feature_ignoreotherprefixesdocupdate.md

```diff
@@ -0,0 +1,5 @@
+### Updated docs for ignore_other_prefixes to clarify behavior ([PR #5592](https://github.com/apollographql/router/pull/5592))
+
+Updated docs for ignore_other_prefixes to clarify behavior
+
+By [@andrewmcgivery](https://github.com/andrewmcgivery) in https://github.com/apollographql/router/pull/5592
```

Comment:
We're in the midst of trying to standardize [guidelines](https://docs.google.com/document/d/1jZhBYPgoZsxe5UEMD3oaGHISoxo6DFR7oJxNij0i6X8/edit#heading=h.2xwuofo9asc4) for changesets, but will mostly likely use imperative for titles

```suggestion
### Update docs for `ignore_other_prefixes` to clarify behavior ([PR #5592](https://github.com/apollographql/router/pull/5592))

Update docs for `ignore_other_prefixes` to clarify behavior

By [@andrewmcgivery](https://github.com/andrewmcgivery) in https://github.com/apollographql/router/pull/5592
```